include/openthread/nat64.h

```diff
@@ -174,6 +311,28 @@ typedef void (*otNat64ReceiveIp4Callback)(otMessage *aMessage, void *aContext);
  */
 void otNat64SetReceiveIp4Callback(otInstance *aInstance, otNat64ReceiveIp4Callback aCallback, void *aContext);
 
+/**
+ * Gets the configured IPv4 CIDR in the NAT64 translator.
+ *
+ * This function is available only when `OPENTHREAD_CONFIG_NAT64_TRANSLATOR_ENABLE` is enabled.
```

Comment:
Remove `This function` -> `Available when` for consistency w/other methods/functions.